app/utils/documents-filter.js

```diff
@@ -0,0 +1,45 @@
+import { tracked } from '@glimmer/tracking';
+
+// state of the documents filter
+export default class DocumentsFilter {
+  @tracked
+  documentName;
```

Comment:
In lijn met andere code deze op 1 lijn zetten `@tracked documentName;`

Reply:
Opgelost in https://github.com/kanselarij-vlaanderen/kaleidos-frontend/pull/839/commits/b8a7657a614f5eeba8e6348b53b346525b69fb47